charts/posthog/templates/clickhouse_instance.yaml

```
@@ -85,6 +85,8 @@ spec:
             fsGroup: 101
           containers:
             - name: clickhouse
+              # KEEP CLICKHOUSE-SERVER VERSION IN SYNC WITH
+              # https://github.com/PostHog/posthog/edit/master/ee/docker-compose.ch.yml#L17
```

```suggestion
              # https://github.com/PostHog/posthog/tree/master/ee/docker-compose.ch.yml#L17
```

maybe? Unless you wanted to explicitly link in edit mode, which I'm not sure if is available for everyone